pkg/api/orchestrators.go

```diff
@@ -149,6 +149,19 @@ func kubernetesInfo(csOrch *OrchestratorProfile) ([]*OrchestratorVersionProfile,
 			return nil, fmt.Errorf("Kubernetes version %s is not supported", csOrch.OrchestratorVersion)
```

Comment:
Is this code redundant now?

Reply:
@tariq1890 I think you can remove the check for `"<1.6.0"` since those will return false for !isVersionSupported(csOrch)